src/components/FetchTrial.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+
+class ListProducers extends Component{
+    constructor(){
+        super();
+        this.state = {
+            producers: []
+        };
+    }
+
+    componentDidMount(){
+        fetch('http://localhost:4000/products')
```

Comment:
all data fetching should handled by triggering an action that runs a saga. So on `componentDidMount` you should trigger something like this:
```
componentDidMount () {
   actions.fetchProducts();
}
```